sig-instrumentation/annual-report-2024.md

```diff
@@ -30,55 +43,54 @@
       Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
 -->
 
-<!-- 
-  - Alpha
-    - [4827 - Component Statusz](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/4827-component-statusz) - v1.32
-    - [4828 - Component Flagz](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/4828-component-flagz) - v1.32
+- Alpha
+  - [4827 - Component Statusz](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/4827-component-statusz) - v1.32
+  - [4828 - Component Flagz](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/4828-component-flagz) - v1.32
 
-  - Beta
-    - [3077 - Contextual logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging) - v1.30
+- Beta
+  - [3077 - Contextual logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging) - v1.30
 
-  - Stable
-    - [2305 - Dynamic Cardinality Enforcement](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2305-metrics-cardinality-enforcement) - v1.30
-    - [647 - APIServer Tracing](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/647-apiserver-tracing) - v1.30 -->
+- Stable
+  - [2305 - Dynamic Cardinality Enforcement](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2305-metrics-cardinality-enforcement) - v1.30
 
 ## [Subprojects](https://git.k8s.io/community/sig-instrumentation#subprojects)
 
+**New in 2024:**
+
+- [zpages](https://git.k8s.io/kubernetes/staging/src/k8s.io/component-base/zpages)
```

Comment:
Also created https://github.com/kubernetes/community/pull/8328 to add this entry

Reply:
I'm not sure if `zpages` as a feature should be put in the same category as, say, UMC or KSM? cc @dashpole @dgrisonnet

Reply:
Let's drop this change then, and revert https://github.com/kubernetes/community/pull/8328?

Reply:
oh my thinking was that since we have "metrics", "structured-logging" listed here already as sub-projects which dont each have their own repo, so maybe it was ok to list zpages as a separate entry here. But I see even traces is not mentioned here

Reply:
Continued in https://github.com/kubernetes/community/pull/8327#discussion_r1970473229

Reply:
Ok, removed zpages entry from subprojects. Kept the list suggested in https://github.com/kubernetes/community/pull/8327#discussion_r1970473229